src/tools/cephfs/cephfs-shell

```diff
@@ -405,12 +408,21 @@ class CephFSShell(Cmd):
                 self.set_prompt()
             return res
         except ConnectionError as e:
-            self.perror('***\n{}'.format(e), end='\n', apply_style=True)
+            perror('***\n{}'.format(e))
+            self.exit_code = 3
         except KeyboardInterrupt:
-            self.perror('Command aborted', end='\n', apply_style=True)
+            perror('Command aborted')
+            self.exit_code = 130
+        except libcephfs.Error as e:
+            perror(e.strerror)
```

Comment:
`perror(e)`

cmd2 will show that it's an exception and print the exception using `str(e)`:

````
CephFS:~/>>> mkdir foo/bar
EXCEPTION of type 'ObjectNotFound' occurred with message: 'error in mkdir foo/bar: No such file or directory [Errno 2]'
To enable full traceback, run the following command:  'set debug true'
````

Which looks fine to me. This also allows us to have future exceptions that do not have an errno but **do** have a `__str__` method.